src/main/java/io/cryostat/net/SslConfiguration.java

```diff
@@ -64,9 +64,12 @@ public class SslConfiguration {
         this.logger = logger;
 
         if (env.hasEnv(Variables.DISABLE_SSL)) {
-            strategy = new NoSslStrategy();
-            logger.info("Selected NoSSL strategy");
-            return;
+            String disableSslValue = env.getEnv(Variables.DISABLE_SSL);
+            if ("true".equalsIgnoreCase(disableSslValue) || "TRUE".equalsIgnoreCase(disableSslValue)) {
```

Comment:
What does `equalsIgnoreCase` do? Do you need to check for both `true` and `TRUE` when you use it?

Reply:
I dont need, I know ignoreCase takes cares of all those cases.. another way i can maintain the consistency is that I can may be convert the var toLowerCase()

Reply:
something like ```    if ("true".equalsIgnoreCase(disableSslValue.toLowerCase())) {
```

Reply:
so this will change any entry TRUE, true, TRue, to true and then makes the behavior consistent?